Source/JavaScriptCore/llint/InPlaceInterpreter64.asm

```diff
@@ -24,31 +24,31 @@
 # Callee save
 
 macro saveIPIntRegisters()
+    # NOTE: We intentionally don't restore memoryBase and boundsCheckingSize here. These are saved
+    # and restored when entering Wasm by the JSToWasm wrapper and changes to them are meant
+    # to be observable within the same Wasm module.
```

Comment:
What happens when calling a wasm function from instanceA from a wasm function from instanceB?
They will have different wasmInstance, memoryBase, and boundsCheckingSize.

Reply:
(resolved offline; this state is handled correctly between the wrapper thunks and IPInt code)